client/css/style.css

```diff
@@ -1450,6 +1450,11 @@ button.collapse-network:first-child:nth-last-child(3) {
 	max-width: 100%;
 }
 
+#chat video {
+	max-width: 640px;
```

Comment:
I'm assuming this width is intentional? Does it max out to 100% on mobile?

Reply:
Yes.